app/src/main/res/values/strings.xml

```diff
@@ -565,4 +565,8 @@ be temporary, and you can try again later.</li>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
     <string name="onboarding_turbo_mode_button_on">Keep Turbo Mode Enabled</string>
     <string name="onboarding_turbo_mode_button_off">Turn Turbo Mode Off</string>
+
+    <!-- Notification shown when saving/unsaving sites, which will be displayed on the homescreen as tiles -->
+    <string name="notification_pinned_site">Pinned to Firefox TV homescreen</string>
```

Comment:
nit: I'm concerned ppl will confuse this with, "Fire TV homescreen"

Reply:
Not sure what we can do about that...? @BrianNJones gave me the string and I don't know how else we could be more explicit without being incredibly wordy.